apps/test-viewer/src/UiProvidersConfig.tsx

```diff
@@ -44,12 +49,42 @@ interface UiItem {
   createUiItemsProvider: () => UiItemsProvider;
 }
 
+
+const SampleSpatialTree: React.FC = () => {
+  const iModel = useActiveIModelConnection();
+  return <>
+    {iModel && <SpatialContainmentTree
+      iModel={iModel}
+      groupByType={true}
+      setGroupByType={() => { return "group-by-type" }}
+      groupByDiscipline={true}
+      setGroupByDiscipline={() => { return "group-by-discipline" }}
+      displayGuids={true}
+      setIsDisplayGuids={() => { return "discard-guid-from-labe" }}
+      enableVisibility={true}
+      clipHeight={1.2}
+      clipAtSpaces={true}
+    />}
+  </>
+}
+const getSpatialTree = (): SelectableContentDefinition => {
```

Comment:
I think this function can be removed and object returned by this function can be inlined in `additionalTrees` array.

Reply:
changed to inline